flak-tests/src/test/java/flask/test/TwoAppsLoginTest.java

```diff
@@ -35,7 +37,7 @@ public class TwoAppsLoginTest {
   @Before
   public void setUp() throws Exception {
     AppFactory fac = TestUtil.getFactory();
-    fac.setPort(9191);
```

Comment:
The PR looks good.

I'm wondering what's the reason behind those changes in tests: is it no longer possible to set a port that way? or is it to avoid setting a hardcoded port? If that is the case, wouldn't it be possible to just call setPort(0)?

Reply:
`setPort` still works, I just update tests to use a random, OS-assigned port, and the loopback IP address. If you don't want to use a loopback address, `setPort(0)` is fine :+1: (actually `setPort` just keep the last IP address defined via `setLocalAddress`)